art/project.py

```diff
@@ -215,14 +218,14 @@ def run_all(self, force_rerun=False):
                 self.fill_step_states(step)
                 step.save_to_disk()
 
-            self.print_summary()
+            self.print_summary(changed_steps)
```

Comment:
Same here I wouldn't require parameter to be passed. Print summary can be aware of changed steps